### DISCUSSION TITLE
Font weight setting

### DISCUSSION THREAD

**Discussion by refact0r** [monkeytypegame/monkeytype#1406] [Ideas]
I think this is pretty self explanatory. Just a slider like the custom background filter that would allow you to change the font weight.

**Comment by frogfrogfrogfrogfrog** [comment]
I like this idea for extremely thick fonts or thin fonts. Although couldn't you use the custom fonts to put in your own font weights?

**Reply by refact0r** [reply]
I think for some fonts you cant.
